screens/HomeScreen.js

```diff
@@ -111,14 +165,43 @@ export default class HomeScreen extends React.Component {
               style={styles.signOutButton}
             />
           </View>
-          <View style={styles.signOutContainer}>
-            <Button
-              title="Go to Camera"
-              onPress={() => this.props.navigation.navigate('Camera')}
-              style={styles.signOutButton}
-            />
-          </View>
+          { this.state.updates ?
+            <View style={styles.signOutContainer}>
+              <Text>New transaction noted, upload your receipt!</Text>
+              <Button
+                title="Upload Receipt"
+                onPress={() => this.props.navigation.navigate('Camera', {
+                  transactionId: this.state.latestTransaction,
+                  customerId: this.state.userId
+                })}
+                style={styles.signOutButton} />
+            </View>
+            : <Text></Text>
+          }
         </ScrollView>
+        <View style={styles.tabBarInfoContainer}>
+          <Text style={styles.tabBarInfoText}>
+            Your transaction history:
+          </Text>
+          { this.state.transactions ? 
+            this.state.transactions.splice(0, 3).map(transaction => {
+              return(
+                <View
+                  key={transaction.get("transaction_id")}
```

Comment:
Nit: probably cleaner if we transform the transaction objects by mapping with a function as they come in from Airtable -- see #11 `createStoreData` etc in `storeHelpers.js`

Reply:
Done!